analyzers/tests/SonarAnalyzer.UnitTest/TestCases/NonDerivedPrivateClassesShouldBeSealed.CSharp11.cs

```diff
@@ -1,10 +1,10 @@
 ﻿using System;
 
-file class ClassNotExtended { } // Noncompliant {{File-scoped classes which are not derived in the current file should be marked as 'sealed'.}}
+file class ClassNotExtended { }             // Noncompliant {{File-scoped classes which are not derived in the current file should be marked as 'sealed'.}}
```

Comment:
Some clean as you code whitespace. The `file` below is the actual change, because of invalid syntax in the new version